statusboard/src/pages/Projects/Projects.tsx

```diff
@@ -204,17 +211,28 @@ function Projects(): JSX.Element {
               options={Object.keys(ACTIVE_THRESHOLDS)}
             />
             <Checkbox
-              label="Display non-brigade projects?"
+              label="Display non Code For America projects?"
               id="non_brigade_projects"
               onChange={(e: ChangeEvent<HTMLInputElement>) =>
                 setFilters({ nonCfA: String(e.target.checked) })
               }
```

Comment:
I'd be ok with changing the default value to display all projects, in which case the text would read "Display only Code for America projects".

What do you think?

Reply:
I'll take another look at my notes from Wed meeting and changing this accordingly.

Reply:
Changed.